src/resources/postcss/customizer-controls.pcss

```diff
@@ -0,0 +1,304 @@
+/* = Customizer Controls (V2)
+============================ */
+
+.wp-customizer {
```

Comment:
there are a lot of values we should be using variables here for, especially color, could we maybe import the variables and mixins and use those?

Reply:
Hm, maybe - I was trying to keep this as "light" as possible for now. Let me see what I can do.